test/instrumentations/prisma/mysql_app/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "lumigo-prisma-mysql-test",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "clean": "rm -f prisma_app.js && rm -rf prisma/migrations",
+    "copy": "cp ../app_shared_resources/prisma_app.js .",
```

Comment:
npm doesn't like referencing the shared app directly from a different folder, so we have to copy it in. but i don't want anyone working on this to be confused as to which app is the real app, so deleting it before and after, copying it in for the run itself, and gitignoring it